### ISSUE TITLE
Could we find a space for rOpenSci in the guide?

### ISSUE THREAD

**Issue by maelle** [sellorm/field-guide-to-the-r-ecosystem#4]
[rOpenSci](https://ropensci.org/) is a community of researchers and developpers developping tools for open science and reproducibility. Some packages are developped by staff (e.g. `magick`) and others contributed by the community (e.g. `ropenaq`).

It could be sorted into community I guess, for instance it has a [discussion forum](https://discuss.ropensci.org/).

It also has a [reviewing system for packages contributed by the community](https://github.com/ropensci/onboarding). I selfishly would like this to be included in order to get submissions of packages [fitting in our categories](https://github.com/ropensci/onboarding/blob/master/policies.md) and to see [people volunteering as reviewers](https://github.com/ropensci/onboarding#why-review).

Poke @stefaniebutland

**Comment by sellorm** [comment]
Yep, rOpenSci should definitely be in there. Possibly something under "Chapter 3 The package ecosystem"?

**Comment by stefaniebutland** [comment]
Hi Mark,
I'm rOpenSci's Community Manager. Thank you for your quick agreement to having something included. I'll have a look this week. Cheers.

and thanks @maelle for the suggestion and poke

**Comment by stefaniebutland** [comment]
@sellorm I'm proposing text for two places - one in package ecosystem and one in community. Tried to fit your tone and length. Let me know what you think.

3.1 Extension Packages and CRAN
(propose to add after BioConductor paragraph)
[rOpenSci](https://ropensci.org/) is a community of researchers and developers making tools for open science and reproducibility. The [rOpenSci package ecosystem](https://github.com/ropensci) is developed by staff as well as by community members through submission to their [open software review system](https://github.com/ropensci/onboarding). Many rOpenSci packages are also available on CRAN.

5.2 Where Can You Find the R Community Online?
- Users and developers can get involved in the [rOpenSci community](https://ropensci.org/community/) in many ways, from reading their blog to submitting a package for open peer review

**Comment by maelle** [comment]
@stefaniebutland isn't this packages page https://ropensci.org/packages/ better for a first contact? (this is actually how _I_ discovered rOpenSci :joy: ) -- I mean for "package ecosystem"

**Comment by stefaniebutland** [comment]
You're right @maelle.

**Comment by stefaniebutland** [comment]
@sellorm Let me know if you want me to submit as pull request

**Comment by sellorm** [comment]
I don't have any issues with the proposed addition to 3.1 other than the location. I'm not sure that's the right place for it. rOpenSci isn't just about packages right? It's much larger than that, with the community elements, blogging, the unconf etc. I'm wondering if we actually need a section in community specifically for rOpenSci. Bioconductor is similar, but I think most users will be aware of it through it's CRAN-like system, which rOpenSci does not have. I'm wondering if combining your two suggestions into a single section in 'Community'. So a new 5.2 specifically about rOpenSci, pushing the current 5.2 to 5.3. 
  
Or a new 5.2 'Community groups' or similar, with rOpenSci and more info about Bioconductor.

What do you think?

**Comment by stefaniebutland** [comment]
(forgive me - I can't seem to use fewer words 😉 )

You've got the right idea about rOpenSci. We're trying to cultivate a *way* of doing things as much as doing things. The challenge you've identified is that there are different ways to present the info. Some people will look to the Field Guide just for "where do I look for packages" and stop before they get to Community.

In Chapter 3 The package ecosystem, I think it's still good to point out Bioconductor and rOpenSci (are there others?) as sources of packages that don't 100% overlap with CRAN, noting their scope. rOpenSci package [scope is outlined here](https://github.com/ropensci/onboarding/blob/master/policies.md). However, you're right that rOpenSci doesn't really fit under the title "3.1 Extension Packages and CRAN". One thing that distinguishes rOpenSci is our peer review system, which is now linked to journals like [JOSS](http://joss.theoj.org/) and [Methods in Ecology and Evolution](https://methodsblog.wordpress.com/2017/11/29/software-review/).

For Community, the titles you have fit well as 5.1 in-person vs 5.2 online. Maybe 5.3 could be a place for communities with a blended model of in-person and online with a short paragraph each about Bioconductor and rOpenSci. (hard to avoid repetition though)

btw, I love that the Field Guide is so concise that I could read the whole thing quickly to get an overview. I checked the bulleted list of where to connect online and it matched a list that someone had put together at our unconf.

I'm happy to fit in to whatever structure you decide.
